test/integration/kubernetes_deploy_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
+  def test_basic
+    runner = KubernetesDeploy::Runner.new(
+      namespace: @namespace,
+      current_sha: "abcabcabc",
+      context: "minikube",
```

Comment:
I'll move it to a constant. I'd like to leave extra helper creation for the follow-up PR though. That's where I've been playing around with approaches for making test setup easy and readable.